src/devTools.js

```diff
@@ -95,13 +101,31 @@ class DevToolsEnhancer {
     }
   }
 
+  importPayloadFrom = (state) => {
+    const instance = {
+      name: this.instanceName,
+      id: this.appInstanceId,
+      store: this.store,
+    };
+    try {
+      const nextLiftedState = importState(state, instance);
+      if (!nextLiftedState) return;
```

Comment:
Nit: can we match the style in our repo? Ie. curly braces, and `return;` on its own line

Reply:
sure